sklearn/metrics/cluster/supervised.py

```diff
@@ -625,8 +625,8 @@ def mutual_info_score(labels_true, labels_pred=None, contingency=None):
                                              sparse=True)
     else:
         warnings.warn("Contingency is deprecated in 0.19 and will be removed" +
-                      "in 0.21 to use contingency matrix pass only contingency"
-                      + " matrix to scoring function", DeprecationWarning)
+                      "in 0.21 to use contingency matrix pass only contingen" +
+                      "cy matrix to scoring function", DeprecationWarning)
```

Comment:
I have checked that before my last commit there were two build jobs out of four had failed (including flakes8 test). But after the last commit one failed job have passed but the job which is not passed is actually caused by assertion error in test_supervised.py file at line 55.

I have got that in job log in Travis CI stated as--

    File "/home/travis/build/scikit-learn/scikit-learn/sklearn/metrics/cluster/tests
    /test_supervised.py", line 55, in test_perfect_matches

    assert_equal(score_func([0., 1., 2.], [42., 7., 2.]), 1.0)

    AssertionError: -inf != 1.0

    '-inf != 1.0' = '%s != %s' % (safe_repr(-inf), safe_repr(1.0))

    '-inf != 1.0' = self._formatMessage('-inf != 1.0', '-inf != 1.0') 

But all test are passing on my system.
Can you gave any suggestion on this? 
Thanks